src/domain/ideas/templates/advancedBoards.tpl.php

```diff
@@ -290,370 +305,421 @@ function tilt_direction(item) {
+                                            <span class="iconfa-comments"></span> <?php echo $row["commentCount"] ?> <?php echo $this->__("text.comments") ?>
+                                            <br/><?php echo $this->__("text.last_modified_on") ?> <?php echo date_format(new DateTime($row["modified"]), "m/d/Y"); ?>
+                                            <?php if ($row['milestoneHeadline'] != '') { ?>
+                                                <br/>
+                                                <hr/>
+                                                <div class="row">
+
+                                                    <div class="col-md-5">
+                                                        <?php echo substr($row['milestoneHeadline'], 0, 10); ?>[...]
+                                                    </div>
+                                                    <div class="col-md-7" style="text-align:right">
+                                                        <?php echo $row['percentDone']; ?><?php echo $this->__("text.percent_complete") ?>
                                                     </div>
-                                                    <div class="row">
-                                                        <div class="col-md-12">
-                                                            <div class="progress">
-                                                                <div class="progress-bar progress-bar-success" role="progressbar" aria-valuenow="<?php echo $row['percentDone']; ?>" aria-valuemin="0" aria-valuemax="100" style="width: <?php echo $row['percentDone']; ?>%">
-                                                                    <span class="sr-only"><?php echo $row['percentDone']; ?>% Complete</span>
-                                                                </div>
+                                                </div>
+                                                <div class="row">
+                                                    <div class="col-md-12">
+                                                        <div class="progress">
+                                                            <div class="progress-bar progress-bar-success"
+                                                                 role="progressbar"
+                                                                 aria-valuenow="<?php echo $row['percentDone']; ?>"
+                                                                 aria-valuemin="0" aria-valuemax="100"
+                                                                 style="width: <?php echo $row['percentDone']; ?>%">
+                                                                <span class="sr-only"><?php echo $row['percentDone']; ?><?php echo $this->__("text.percent_complete") ?></span>
```

Comment:
In cases like X% complete I think we should use sprintf. The order of words could be different in other languages eg Complete X%. Example is in https://github.com/Leantime/leantime/blob/feature-i18nSupport/src/domain/dashboard/templates/show.tpl.php (line 467) 

`><?=sprintf($this->__("text.percent_complete"), $percent)?>`

Reply:
Also (I just found that out) there is a difference in spacing before the % sign for different languages: https://en.wikipedia.org/wiki/Percent_sign

Reply:
Fixed